apps/meteor/ee/server/local-services/reads/service.ts

```diff
@@ -0,0 +1,50 @@
+import { Reads, Subscriptions } from '@rocket.chat/models';
+import type { ISubscription } from '@rocket.chat/core-typings';
+import { ServiceClassInternal } from '@rocket.chat/core-services';
+
+import type { IReadsService } from '../../sdk/types/IReadsService';
+import { Messages } from '../../../../app/models/server';
+import { ReadReceipt } from '../../lib/message-read-receipt/ReadReceipt';
+import { MAX_ROOM_SIZE_CHECK_INDIVIDUAL_READ_RECEIPTS } from '../../lib/constants';
+
+export class ReadsService extends ServiceClassInternal implements IReadsService {
```

Comment:
A bit picky, but could we name this `ReadReceipts`? 
I know `Reads` makes sense when you have the context, but otherwise it's not very descriptive

Reply:
I do agree that `Reads` is not a meaningful name, but I also think that calling it `ReadReceipts` could be misleading :thinking: since it manages the `Reads` collection (which currently works like subscriptions to threads, but we plan to expand it)

Reply:
Just out of curiosity, why we needed another collection for thread subscriptions? 👀

Reply:
That's because we decided to treat read receipts for messages in threads just as if threads were "separate rooms". So in order to control which messages have been read, we gotta know whether everyone in the room opened the thread and the messages they read.
For messages in the main room, we use subscriptions' `ls` field (`lastSeen`), so that we can get the minimum value of this field among room members and mark all older messages as read. We needed something similar for threads.

Reply:
what do you guys think of calling the service as `MessageReads`? a bit more meaningful I guess 😬

Reply:
hmmm sound better to me :thinking: it's currently just "threads' reads", but it's better for us not to be so specific, I guess
I'll update the service name

Reply:
Done! Updated the service name to `MessageReads` and the collection name to `message_reads`